lib/src/util/class_names.dart

```diff
@@ -102,8 +105,10 @@ class ClassNameBuilder {
     _classNamesBuffer.write(className);
   }
 
-  /// Adds all of the CSS classes represented by [className] (a space-delimited list) to the blacklist,
-  /// IF [should] is true, otherwise, does nothing (convenience for helping to inline blacklisting conditionals).
+  /// Adds all of the CSS classes represented by [className] _(a space-delimited list)_ to the [_blacklistBuffer] -
```

Comment:
`to the [_blacklistBuffer] ` is an irrelevant (and possibly confusing) implementation detail (and can be seen via the dartdoc or by clicking into the source, if desired).